backend/app/api/v1/breadth.py

```
@@ -63,6 +71,38 @@ def _require_task_controls() -> None:
         )
 
 
+@router.get(
+    "/contributors/index",
+    response_model=BreadthContributorIndexResponse,
+)
+async def get_breadth_contributor_index(
+    market: str = Query("US", description=MARKET_QUERY_DESCRIPTION),
+    db: Session = Depends(get_db),
```

Ruff (0.16.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)